craft_cli/printer.py

```diff
@@ -87,9 +109,8 @@ def _format_term_line(prefix: str, text: str, spintext: str, *, ephemeral: bool)
             text = text[-remaining_for_last_line:]
             if len(text) > usable:
                 text = text[: usable - 1] + "…"
-    cleaner = " " * (usable - len(text) % width)
 
-    return prefix + text + spintext + cleaner
+    return prefix + fill_line(text + spintext)
```

Comment:
don't you need to take the prefix into account, for the non-`ANSI_CLEAR_LINE_TO_END` case?

Reply:
Actually I stand corrected. The `prefix` here is the previous line's end. I've renamed it for less ambiguity.